packages/block-library/src/embed/wp-embed-preview.js

```diff
@@ -38,15 +70,21 @@ class WpEmbedPreview extends Component {
 
 	render() {
 		const { html } = this.props;
-		const doc = new DOMParser().parseFromString( html, 'text/html' );
-		doc.querySelector( 'iframe' ).removeAttribute( 'style' );
-		doc.querySelector( 'blockquote' ).style.display = 'none';
+		if ( this.node.current ) {
```

Comment:
In terms of life cycle, is this the right place to do this?

What about using callback refs like:

```js
function processContents( ref ) {
  if ( ref ) {
    ref.querySelector( ... ).style.display = 'none';
  ...
}

<div ref={ processContents }>...</div>